src/Miunie.Discord/Convertors/MiunieUserConvertor.cs

```diff
@@ -0,0 +1,42 @@
+using System.Threading.Tasks;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Converters;
+using DSharpPlus.Entities;
+using Miunie.Core;
+
+namespace Miunie.Discord.Convertors
+{
+    public class MiunieUserConvertor : IArgumentConverter<MiunieUser>
+    {
+        private readonly DiscordMemberConverter _dmConverter;
+        private readonly MiunieUserService _userService;
+
+        public MiunieUserConvertor(MiunieUserService userService)
+        {
+            _dmConverter = new DiscordMemberConverter();
+            _userService = userService;
+        }
+
+        public async Task<Optional<MiunieUser>> ConvertAsync(
+                                                    string userInput,
+                                                    CommandContext context)
```

Comment:
Unfortunate. This code smell is that the type is unnecessarily long.
But again, we inherit this from the Library and therefore don't have control over it.
:man_shrugging: